### DISCUSSION TITLE
Welcome to nmdc-edge Discussions!

### DISCUSSION THREAD

**Discussion by shreddd** [microbiomedata/nmdc-edge#108] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eecavanna** [comment]
Hi, I'm @eecavanna and I'll be working with @mflynn-lanl to deploy `nmdc-edge` to Jetstream2. 

My previous work in this repository has consisted of **container**-izing the web app (it's a Node.js app that serves a React.js client and services HTTP requests made by that same client), **documenting** the repository as a whole (in `README.md`), and standardizing **environment variables**. Aside from that, I'm not too familiar with the code base (i.e. specific files, functions, variables). I also haven't used Cromwell before.

I have experience using AWS (e.g. S3, RDS, ECR, ECS via Fargate, Secrets Manager), Digital Ocean (virtual private servers), Cloudflare Workers (serverless functions), and a little bit with GCP (e.g. GCE, GKE, and Stackdriver); mostly for deploying web apps.
